<!DOCTYPE html>
<html class="client-nojs" lang="en" dir="ltr">
<head>
<meta charset="UTF-8"/>
<title>SPIM Registration Method - ImageJ</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"SPIM_Registration_Method","wgTitle":"SPIM Registration Method","wgCurRevisionId":13723,"wgRevisionId":13723,"wgArticleId":895,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["Pages with reference errors","Plugins","Registration"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"SPIM_Registration_Method","wgRelevantArticleId":895,"wgRequestId":"a3a930fe8ed5fdebe6849b85","wgIsProbablyEditable":false,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgPreferredVariant":"en","fancytree_path":"/extensions/TreeAndMenu/fancytree"});mw.loader.state({"site.styles":"ready","noscript":"ready","user.styles":"ready","user.cssprefs":"ready","user":"ready","user.options":"loading","user.tokens":"loading","ext.cite.styles":"ready","ext.math.styles":"ready","mediawiki.legacy.shared":"ready","mediawiki.legacy.commonPrint":"ready","mediawiki.sectionAnchor":"ready","skins.erudite":"ready"});mw.loader.implement("user.options@0j3lz3q",function($,jQuery,require,module){mw.user.options.set({"variant":"en"});});mw.loader.implement("user.tokens@1ku9xth",function ( $, jQuery, require, module ) {
mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});/*@nomin*/;

});mw.loader.load(["mediawiki.page.startup"]);});</script>
<link rel="stylesheet" href="load.php%3Fdebug=false&amp;lang=en&amp;modules=ext.cite.styles%257Cext.math.styles%257Cmediawiki.legacy.commonPrint%252Cshared%257Cmediawiki.sectionAnchor%257Cskins.erudite&amp;only=styles&amp;skin=erudite.css"/>
<script async="" src="load.php%3Fdebug=false&amp;lang=en&amp;modules=startup&amp;only=scripts&amp;skin=erudite"></script>
<link rel="stylesheet" href="extensions/TreeAndMenu/fancytree/fancytree.css"/><link rel="stylesheet" href="extensions/TreeAndMenu/suckerfish/suckerfish.css"/>
<meta name="ResourceLoaderDynamicStyles" content=""/>
<link rel="stylesheet" href="load.php%3Fdebug=false&amp;lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=erudite.css"/>
<meta name="generator" content="MediaWiki 1.28.0"/>
<meta name="description" content="Please note that the SPIM registration plugin available through Fiji, is based on a publication. If you use it successfully for your research please be so kind to cite our work:"/>
<link rel="shortcut icon" href="skins/ij2.ico"/>
	<meta property="og:type" content="article"/>

	<meta property="og:site_name" content="ImageJ"/>

	<meta property="og:title" content="SPIM Registration Method"/>

	<meta property="og:description" content="Please note that the SPIM registration plugin available through Fiji, is based on a publication. If you use it successfully for your research please be so kind to cite our work:"/>


<meta name="viewport" content="width=device-width, initial-scale=1" />
</head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject page-SPIM_Registration_Method rootpage-SPIM_Registration_Method skin-erudite action-view">
		<div class="mw-jump">
			<a href="SPIM_Registration_Method.html#bodyContent">Skip to content</a>, 			<a href="SPIM_Registration_Method.html#search">Skip to search</a>
		</div>

		<div id="top-wrap" role="banner">
			<h1><a href="Welcome" title="ImageJ" rel="home">ImageJ</a></h1>
			<div id="tagline">From ImageJ</div>

			<a id="menubutton" href="SPIM_Registration_Method.html#menu">Menu</a>
			<div id="nav" role="navigation">
			<ul id='menu'>
<li id="menu-item-n-About"><a href="ImageJ">About</a></li>
<li id="menu-item-n-Downloads"><a href="Downloads">Downloads</a></li>
<li id="menu-item-n-Learn"><a href="Learn">Learn</a></li>
<li id="menu-item-n-Develop"><a href="Development">Develop</a></li>
<li id="menu-item-n-News"><a href="News">News</a></li>
<li id="menu-item-n-Events"><a href="Events">Events</a></li>
<li id="menu-item-n-Help"><a href="Help">Help</a></li>
</ul>
			</div>
		</div>

		<div id="mw-js-message"></div>
		
		<div id="main" role="main">

			<div id="bodyContent">
				<h1>SPIM Registration Method</h1>
				
				<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div id="toc" class="toc"><div id="toctitle"><h2>Contents</h2></div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="SPIM_Registration_Method.html#Citation"><span class="tocnumber">1</span> <span class="toctext">Citation</span></a></li>
<li class="toclevel-1 tocsection-2"><a href="SPIM_Registration_Method.html#Important_Note"><span class="tocnumber">2</span> <span class="toctext">Important Note</span></a></li>
<li class="toclevel-1 tocsection-3"><a href="SPIM_Registration_Method.html#Introduction"><span class="tocnumber">3</span> <span class="toctext">Introduction</span></a>
<ul>
<li class="toclevel-2 tocsection-4"><a href="SPIM_Registration_Method.html#SPIM_principles"><span class="tocnumber">3.1</span> <span class="toctext">SPIM principles</span></a></li>
<li class="toclevel-2 tocsection-5"><a href="SPIM_Registration_Method.html#Related_work"><span class="tocnumber">3.2</span> <span class="toctext">Related work</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-6"><a href="SPIM_Registration_Method.html#The_Method"><span class="tocnumber">4</span> <span class="toctext">The Method</span></a>
<ul>
<li class="toclevel-2 tocsection-7"><a href="SPIM_Registration_Method.html#Bead_Segmentation"><span class="tocnumber">4.1</span> <span class="toctext">Bead Segmentation</span></a></li>
<li class="toclevel-2 tocsection-8"><a href="SPIM_Registration_Method.html#Establishing_Bead_Correspondences"><span class="tocnumber">4.2</span> <span class="toctext">Establishing Bead Correspondences</span></a></li>
<li class="toclevel-2 tocsection-9"><a href="SPIM_Registration_Method.html#Global_Optimization"><span class="tocnumber">4.3</span> <span class="toctext">Global Optimization</span></a></li>
<li class="toclevel-2 tocsection-10"><a href="SPIM_Registration_Method.html#Time-lapse_registration"><span class="tocnumber">4.4</span> <span class="toctext">Time-lapse registration</span></a></li>
<li class="toclevel-2 tocsection-11"><a href="SPIM_Registration_Method.html#Image_Fusion_and_blending"><span class="tocnumber">4.5</span> <span class="toctext">Image Fusion and blending</span></a></li>
<li class="toclevel-2 tocsection-12"><a href="SPIM_Registration_Method.html#Strategies_for_bead_removal"><span class="tocnumber">4.6</span> <span class="toctext">Strategies for bead removal</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-13"><a href="SPIM_Registration_Method.html#Evaluation"><span class="tocnumber">5</span> <span class="toctext">Evaluation</span></a>
<ul>
<li class="toclevel-2 tocsection-14"><a href="SPIM_Registration_Method.html#Evaluation_of_the_performance_of_the_bead-based_registration_framework"><span class="tocnumber">5.1</span> <span class="toctext">Evaluation of the performance of the bead-based registration framework</span></a></li>
<li class="toclevel-2 tocsection-15"><a href="SPIM_Registration_Method.html#Performance_of_the_bead-based_registration_framework_in_comparison_with_intensity-based_methods"><span class="tocnumber">5.2</span> <span class="toctext">Performance of the bead-based registration framework in comparison with intensity-based methods</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-16"><a href="SPIM_Registration_Method.html#Samples_imaged_by_SPIM"><span class="tocnumber">6</span> <span class="toctext">Samples imaged by SPIM</span></a>
<ul>
<li class="toclevel-2 tocsection-17"><a href="SPIM_Registration_Method.html#Overview_of_imaged_specimens"><span class="tocnumber">6.1</span> <span class="toctext">Overview of imaged specimens</span></a></li>
<li class="toclevel-2 tocsection-18"><a href="SPIM_Registration_Method.html#Sample_mounting_for_SPIM"><span class="tocnumber">6.2</span> <span class="toctext">Sample mounting for SPIM</span></a></li>
<li class="toclevel-2 tocsection-19"><a href="SPIM_Registration_Method.html#Sample_independent_registration"><span class="tocnumber">6.3</span> <span class="toctext">Sample independent registration</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-20"><a href="SPIM_Registration_Method.html#Broad_applicability_of_the_bead-based_framework_to_multi-view_imaging"><span class="tocnumber">7</span> <span class="toctext">Broad applicability of the bead-based framework to multi-view imaging</span></a>
<ul>
<li class="toclevel-2 tocsection-21"><a href="SPIM_Registration_Method.html#Sample_mounting_for_multi-view_imaging_on_an_upright_microscope"><span class="tocnumber">7.1</span> <span class="toctext">Sample mounting for multi-view imaging on an upright microscope</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-22"><a href="SPIM_Registration_Method.html#Implementation"><span class="tocnumber">8</span> <span class="toctext">Implementation</span></a></li>
<li class="toclevel-1 tocsection-23"><a href="SPIM_Registration_Method.html#Acknowledgments"><span class="tocnumber">9</span> <span class="toctext">Acknowledgments</span></a></li>
<li class="toclevel-1 tocsection-24"><a href="SPIM_Registration_Method.html#References"><span class="tocnumber">10</span> <span class="toctext">References</span></a></li>
</ul>
</div>

<h2><span class="mw-headline" id="Citation">Citation</span></h2>
<p>Please note that the SPIM registration plugin available through Fiji, is based on a publication. If you use it successfully for your research please be so kind to cite our work:
</p>
<ul><li> S. Preibisch, S. Saalfeld, J. Schindelin and P. Tomancak (2010) "Software for bead-based registration of selective plane illumination microscopy data", <i>Nature Methods</i>, <b>7</b>(6):418-419. <a rel="nofollow" class="external text" href="http://www.nature.com/nmeth/journal/v7/n6/full/nmeth0610-418.html">Webpage</a> <a href="_images/2/29/Nmeth0610-418.pdf" class="internal" title="Nmeth0610-418.pdf">PDF</a> <a href="_images/3/32/Nmeth0610-418-S1.pdf" class="internal" title="Nmeth0610-418-S1.pdf">Supplement</a></li></ul>
<h2><span class="mw-headline" id="Important_Note">Important Note</span></h2>
<p><span style="color:#A52A2A">
<i><b>For details about the SPIM registration, fusion &amp; deconvolution please have a look at the <a href="Multiview-Reconstruction" title="Multiview-Reconstruction">Multiview Reconstruction Plugin</a>. It is much more powerful, flexible and completely integrated with the <a href="BigDataViewer" title="BigDataViewer">BigDataViewer</a>. Documentation on the outdated <a href="SPIM_Registration" title="SPIM Registration">SPIM Registration</a> is still available.</b></i>
</span>
</p>
<h2><span class="mw-headline" id="Introduction">Introduction</span></h2>
<h3><span class="mw-headline" id="SPIM_principles">SPIM principles</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="./File:SPIMScheme.png" class="image"><img alt="" src="_images/8/8b/SPIMScheme.png" width="300" height="300" class="thumbimage" /></a>  <div class="thumbcaption"><div class="magnify"><a href="./File:SPIMScheme.png" class="internal" title="Enlarge"></a></div><b>Figure&#160;1:</b> Schematic drawing of a Selective Plane Illumination Microscope</div></div></div>
<p>A Selective Plane Illumination Microscope<sup id="cite_ref-HuiskenAl2004_1-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-HuiskenAl2004-1">&#91;1&#93;</a></sup>
(<a href="_images/8/8b/SPIMScheme.png" class="internal" title="SPIMScheme.png">Figure&#160;1</a>), achieves optical
sectioning by focusing the excitation laser into a thin laser light sheet that reaches its minimal thickness in the middle of the field of view.  The light sheet enters the water filled sample chamber and illuminates the sample 
which is embedded in an agarose column. The agarose protrudes from the end of a glass capillary attached to a motor that rotates the sample.
The objective lens is arranged perpendicular to the light sheet. The perpendicular orientation of the illumination and detection optics ensures that only a section of the specimen in-focus is illuminated, minimizing photo-bleaching and laser damage of the living samples and allowing
for very long time-lapse recordings. Two-dimensional images of emitted
fluorescent light are captured by a CCD camera focussed on the center of the light-sheet. The CCD camera captures the light-sheet-illuminated section in a single exposure enabling a very fast acquisition rate important for capturing dynamic developmental events. In order to acquire 3d image stacks, the sample is moved through the light sheet in increments of 0.5&#160;&#956;m to 5&#160;&#956;m depending on the objective and the light sheet thickness.
</p><p>The SPIM instrument can, in principle, achieve an isotropic, high resolution along <img class="mwe-math-fallback-image-inline tex" alt="x" src="_images/math/9/d/d/9dd4e461268c8034f5c8564e155c67a6.png" />, <img class="mwe-math-fallback-image-inline tex" alt="y" src="_images/math/4/1/5/415290769594460e2e485922904f345d.png" /> and <img class="mwe-math-fallback-image-inline tex" alt="z" src="_images/math/f/b/a/fbade9e36a3f36d3d676c1b808451dd7.png" />-axis allowing <i>in toto</i>  imaging of large 3d specimens. In order to achieve an isotropic resolution uniformly across the sample volume in all three dimensions, it is necessary to rotate the sample and record image stacks of the same specimen from different angles (usually 3 to 12, see <a href="_images/3/3f/Supplementary_Video_1_SPIM_in_Action.mov" class="internal" title="Supplementary Video 1 SPIM in Action.mov">Video&#160;1</a>).
</p>
<h3><span class="mw-headline" id="Related_work">Related work</span></h3>
<p>Multi-view microscopy techniques, such as tilted confocal acquisitions,<sup id="cite_ref-ShawAl89_2-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-ShawAl89-2">&#91;2&#93;</a></sup>
SPIM<sup id="cite_ref-HuiskenAl2004_1-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-HuiskenAl2004-1">&#91;1&#93;</a></sup> and Ultramicroscopy<sup id="cite_ref-Dodt2007_3-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Dodt2007-3">&#91;3&#93;</a></sup>
(2 views), can increase the resolution along the <img class="mwe-math-fallback-image-inline tex" alt="z" src="_images/math/f/b/a/fbade9e36a3f36d3d676c1b808451dd7.png" />-axis and thus enable the analysis of objects smaller than the axial resolution limit.<sup id="cite_ref-ShawAl89_2-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-ShawAl89-2">&#91;2&#93;</a></sup><sup id="cite_ref-Swoger2007_4-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Swoger2007-4">&#91;4&#93;</a></sup>
Image reconstruction based on the image intensities requires significant overlap of image content which is often difficult to achieve particularly in live imaging of dynamically changing samples.
</p><p>The idea to incorporate fiduciary markers to facilitate sample independent reconstruction is widely used in medical imaging<sup id="cite_ref-Gullekson1974_5-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Gullekson1974-5">&#91;5&#93;</a></sup><sup id="cite_ref-Erickson1993_6-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Erickson1993-6">&#91;6&#93;</a></sup><sup id="cite_ref-Fitzpatrick2001_7-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Fitzpatrick2001-7">&#91;7&#93;</a></sup><sup id="cite_ref-Wiles2008_8-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Wiles2008-8">&#91;8&#93;</a></sup><sup id="cite_ref-Moghari2008_9-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Moghari2008-9">&#91;9&#93;</a></sup>
and electron tomography.<sup id="cite_ref-Dierksen1992_10-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Dierksen1992-10">&#91;10&#93;</a></sup><sup id="cite_ref-Koster1997_11-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Koster1997-11">&#91;11&#93;</a></sup>
Due to the low amount of fiduciary markers available for registration, research is focused on error analysis rather than efficiency of matching of thousands of markers with only partial overlap.<sup id="cite_ref-Fitzpatrick2001_7-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Fitzpatrick2001-7">&#91;7&#93;</a></sup>
</p><p>In contrast, in the robotics and automation field, there is interest in localization of large amounts of different objects. Points of interest are extracted from photos and checked against databases to determine their type and orientation.<sup id="cite_ref-HarrisS88_12-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-HarrisS88-12">&#91;12&#93;</a></sup><sup id="cite_ref-Canny86_13-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Canny86-13">&#91;13&#93;</a></sup>
To enable real time object recognition, Lamdan et al.<sup id="cite_ref-LamdanAl88_14-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-LamdanAl88-14">&#91;14&#93;</a></sup>
introduced &#8216;geometric hashing'&#8217;  which uses an intrinsic invariant local coordinate system to match objects against database entries in a viewpoint independent manner. The geometric hashing principle is reused in the fields of astronomy<sup id="cite_ref-Hogg2008_15-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Hogg2008-15">&#91;15&#93;</a></sup>
and protein structure alignment and comparison<sup id="cite_ref-Nussinov91_16-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Nussinov91-16">&#91;16&#93;</a></sup><sup id="cite_ref-Fischer94_17-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Fischer94-17">&#91;17&#93;</a></sup><sup id="cite_ref-Wallace1997_18-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Wallace1997-18">&#91;18&#93;</a></sup><sup id="cite_ref-Stark2003_19-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Stark2003-19">&#91;19&#93;</a></sup>
where efficient searching in massive point clouds is required.
</p><p>The use of local descriptors instead of complete scenes for matching is proposed in many fields comprising image registration,<sup id="cite_ref-Stanski2005_20-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Stanski2005-20">&#91;20&#93;</a></sup><sup id="cite_ref-Lowe04_21-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Lowe04-21">&#91;21&#93;</a></sup>
robotics and autonomous systems,<sup id="cite_ref-Kuipers91_22-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Kuipers91-22">&#91;22&#93;</a></sup><sup id="cite_ref-Bradley04_23-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Bradley04-23">&#91;23&#93;</a></sup>
and computer vision.<sup id="cite_ref-Frome04_24-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Frome04-24">&#91;24&#93;</a></sup>
</p><p>Matula et al.<sup id="cite_ref-Matula2003_25-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Matula2003-25">&#91;25&#93;</a></sup>
suggest segmentation based approaches for reconstruction of multi-view microscopy images.  The center of mass of the cloud of segmented objects is used as a reference point for a cylindrical coordinate system facilitating the registration between two views.  Similarly to intensity based approaches, this method requires significant overlap between the images and furthermore supports alignment of only two stacks at a time.
</p><p>Our approach combines the idea of using fiduciary markers, local descriptors and geometric hashing and applies global optimization.  It can register an arbitrary number of partially overlapping point clouds.  It is robust with respect to the amount of incorporated beads, bead distribution, amount of overlap, and can reliably detect non-affine disturbances (e.g. abrupt agarose movement) that might occur during imaging (<a href="SPIM_Registration_Method.html#Table1">Table&#160;1</a>).
</p>
<h2><span class="mw-headline" id="The_Method">The Method</span></h2>
<h3><span class="mw-headline" id="Bead_Segmentation">Bead Segmentation</span></h3>
<p>The incorporated sub-resolution beads appear as the Point Spread Function (PSF) of the microscopic system in each image&#160;<img class="mwe-math-fallback-image-inline tex" alt="I(x,y,z)" src="_images/math/7/c/e/7ce41255f85a7f15d119cbce613b5786.png" />. To detect beads, one would ideally convolve it with the impulse response&#160;(PSF) of the microscope yielding highest correlation at the sub-pixel location of each bead. However, the PSF is not constant over different experiments due to changing exposure times, laser power, bead types, objectives and agarose concentration. Furthermore, the PSF is not constant across the field of view due to the concavity of the light sheet and thus the convolution operation is computationally very demanding.  
</p><p>We found that an appropriately smoothed 3d LaPlace filter&#160;<img class="mwe-math-fallback-image-inline tex" alt="\nabla^2" src="_images/math/6/d/d/6dd8035713e67f816cb0b819cf492ffd.png" /> detects all beads with sufficient accuracy while effectively suppressing high frequency noise.  As suggested in the Computer Vision literature,<sup id="cite_ref-Lindeberg94_26-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Lindeberg94-26">&#91;26&#93;</a></sup><sup id="cite_ref-Lowe04_21-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Lowe04-21">&#91;21&#93;</a></sup>
we approximate <img class="mwe-math-fallback-image-inline tex" alt="\nabla^2I" src="_images/math/4/a/8/4a8e17fa403028dfe86d663083fd75ef.png" /> by the difference of two Gaussian convolutions (DoG) of the image&#160;<img class="mwe-math-fallback-image-inline tex" alt="I" src="_images/math/d/d/7/dd7536794b63bf90eccfd37f9b147d7f.png" /> with a standard deviation&#160;<img class="mwe-math-fallback-image-inline tex" alt="\sigma" src="_images/math/9/d/4/9d43cb8bbcb702e9d5943de477f099e2.png" /> of 1.4&#160;px and 1.8&#160;px respectively.  All local minima in a 3&#215;3&#215;3 neighborhood in <img class="mwe-math-fallback-image-inline tex" alt="\nabla^2I" src="_images/math/4/a/8/4a8e17fa403028dfe86d663083fd75ef.png" /> represent intensity maxima whose sub-pixel location is then estimated by fitting a 3d quadratic function to this neighbourhood.<sup id="cite_ref-BrownLowe02_27-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-BrownLowe02-27">&#91;27&#93;</a></sup>
The DoG detector identifies beads even if they are close to each other, close to the sample or those with an unexpected shape.  It also massively oversegments the image detecting &#8216;blob-like&#8217; structures, corners and various locations alongside edges or planes within the imaged sample. However, those detections do not interfere with the registration process as the descriptors that incorporate them are filtered out by local descriptor matching (see <a href="_images/9/94/DescriptorBuildup.png" class="internal" title="DescriptorBuildup.png">Figure&#160;2</a>).  Only beads are repeatably detected in different views.
</p>
<h3><span class="mw-headline" id="Establishing_Bead_Correspondences">Establishing Bead Correspondences</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="./File:DescriptorBuildup.png" class="image"><img alt="" src="_images/9/94/DescriptorBuildup.png" width="300" height="169" class="thumbimage" /></a>  <div class="thumbcaption"><div class="magnify"><a href="./File:DescriptorBuildup.png" class="internal" title="Enlarge"></a></div><b>Figure&#160;2:</b> Rotation invariant local geometric descriptor</div></div></div>
<p>To register two views&#160;<img class="mwe-math-fallback-image-inline tex" alt="A" src="_images/math/7/f/c/7fc56270e7a70fa81a5935b72eacbe29.png" /> and <img class="mwe-math-fallback-image-inline tex" alt="B" src="_images/math/9/d/5/9d5ed678fe57bcca610140957afab571.png" /> the corresponding bead pairs <img class="mwe-math-fallback-image-inline tex" alt="(\vec{a},\vec{b})" src="_images/math/5/6/a/56a686235b248aac541bcbbb94052fd9.png" /> have to be identified invariantly to translation and rotation. To this end, we developed a <i>geometric local descriptor</i> . The local descriptor of a bead is defined by the locations of its 3&#160;nearest neighbors in 3d image space ordered by their distance to the bead. To efficiently extract the nearest neighbors in image space we use the <i>kd</i>-tree implementation of the WEKA framework.<sup id="cite_ref-WEKA_28-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-WEKA-28">&#91;28&#93;</a></sup>
Translation invariance is achieved by storing locations relative to the bead.  That is, each bead descriptor is an ordered 3d point cloud of cardinality 3 with its origin <img class="mwe-math-fallback-image-inline tex" alt="\vec{o} = (0,0,0)^T" src="_images/math/3/f/f/3ff0cae0452bfc921d20758c7ba6724e.png" />} being the location of the bead.
</p><p>Local descriptor matching is performed invariantly to rotation by mapping the ordered point cloud of all beads <img class="mwe-math-fallback-image-inline tex" alt="\vec{a}\in A" src="_images/math/6/5/f/65fe6cf8eddc220a26383e1ea963fa19.png" /> to that of all beads <img class="mwe-math-fallback-image-inline tex" alt="\vec{b}\in B" src="_images/math/c/4/7/c476e095b8e67e79f726c6dfad30a75d.png" /> individually by means of least square point mapping error using the closed-form unit quaternion-based solution.<sup id="cite_ref-Horn87_29-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Horn87-29">&#91;29&#93;</a></sup>
The similarity measure <img class="mwe-math-fallback-image-inline tex" alt="\epsilon" src="_images/math/c/5/0/c50b9e82e318d4c163e4b1b060f7daf5.png" /> is the average point mapping error. Each candidate in <img class="mwe-math-fallback-image-inline tex" alt="A" src="_images/math/7/f/c/7fc56270e7a70fa81a5935b72eacbe29.png" /> is matched against each candidate in <img class="mwe-math-fallback-image-inline tex" alt="B" src="_images/math/9/d/5/9d5ed678fe57bcca610140957afab571.png" />.  Corresponding descriptors are those with minimal <img class="mwe-math-fallback-image-inline tex" alt="\epsilon" src="_images/math/c/5/0/c50b9e82e318d4c163e4b1b060f7daf5.png" />. This approach, however, is computationally very demanding as it has a complexity of <img class="mwe-math-fallback-image-inline tex" alt="O(n^2)" src="_images/math/1/8/9/189317b4b935a745fcfaf95940d2b4f0.png" /> regarding the number of detections.<sup id="cite_ref-Preibisch2009a_30-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Preibisch2009a-30">&#91;30&#93;</a></sup>
</p><p>We therefore employed a variation of geometric hashing<sup id="cite_ref-LamdanAl88_14-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-LamdanAl88-14">&#91;14&#93;</a></sup> to speed up the matching process. Instead of using one reference coordinate system for the complete scene we define a local coordinate system for each of the descriptor as illustrated and described in <a href="_images/9/94/DescriptorBuildup.png" class="internal" title="DescriptorBuildup.png">Figure&#160;2</a>. All remaining bead coordinates not used for defining the local coordinate system become rotation invariant which enables us to compare descriptors very efficiently using <i>kd</i>-trees to index remaining bead coordinates in the local coordinate system.
Again, we employ the <i>kd</i>-tree implementation of the WEKA framework<sup id="cite_ref-WEKA_28-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-WEKA-28">&#91;28&#93;</a></sup> on a six-dimensional tree to identify nearest neighbors in the descriptor space, i.e. descriptors which are most similar. The most similar descriptors that are significantly better (10&#215;) than the second nearest neighbor in descriptor space are designated correspondence candidates.<sup id="cite_ref-Lowe04_21-2" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Lowe04-21">&#91;21&#93;</a></sup>
</p><p>Descriptors composed of only four beads are not completely distinctive and similar descriptors can occur by chance. Increasing the number of beads in the descriptor would make it more distinctive to the cost of less identified correspondences and increased computation time. All true correspondences agree on one transformation model for optimal view registration, whereas each false correspondence supports a different transformation. Therefore, we used the minimal descriptor size (4 beads) and rejected false correspondences from candidate sets with the Random Sample Consensus (RANSAC)<sup id="cite_ref-FischlerB81_31-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-FischlerB81-31">&#91;31&#93;</a></sup>
on the affine transformation model followed by robust regression.
</p>
<h3><span class="mw-headline" id="Global_Optimization">Global Optimization</span></h3>
<p>The identified set of corresponding beads <img class="mwe-math-fallback-image-inline tex" alt="C_{AB} = \{(\vec{a}_i,\vec{b}_i) : i=\{1,2, \dots, \left| C\right|\}\}" src="_images/math/3/1/0/310c84ca995d4f655169f523b6bfa482.png" /> for a pair of views <img class="mwe-math-fallback-image-inline tex" alt="A" src="_images/math/7/f/c/7fc56270e7a70fa81a5935b72eacbe29.png" /> and <img class="mwe-math-fallback-image-inline tex" alt="B" src="_images/math/9/d/5/9d5ed678fe57bcca610140957afab571.png" /> defines an affine transformation&#160;<img class="mwe-math-fallback-image-inline tex" alt="\mathbf{T}_{AB}" src="_images/math/8/f/4/8f4d8575c3e759097995c2bd905838bc.png" /> that maps <img class="mwe-math-fallback-image-inline tex" alt="A" src="_images/math/7/f/c/7fc56270e7a70fa81a5935b72eacbe29.png" /> to <img class="mwe-math-fallback-image-inline tex" alt="B" src="_images/math/9/d/5/9d5ed678fe57bcca610140957afab571.png" /> by means of least square bead correspondence displacement
</p>
<center><img class="mwe-math-fallback-image-inline tex" alt="&#10;\arg\min_{\mathbf{T}_{AB}}\sum_{(\vec{a},\vec{b})\in C_{ab}}{\left\|\mathbf{T}_{AB}\vec{a} - \vec{b}\right\|^2}.&#10;" src="_images/math/8/7/f/87f1cc192cdff903702f50f80c404f03.png" /></center>
<p>We use an affine transformation to correct for the anisotropic <img class="mwe-math-fallback-image-inline tex" alt="z" src="_images/math/f/b/a/fbade9e36a3f36d3d676c1b808451dd7.png" />-stretching of each view introduced by the differential refraction index mismatch between water and agarose<sup id="cite_ref-Hell1993_32-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Hell1993-32">&#91;32&#93;</a></sup><sup id="cite_ref-Matula2003_25-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Matula2003-25">&#91;25&#93;</a></sup> as the sample is never perfectly centered in the agarose column.
</p><p>Registration of more than two views requires groupwise optimization of the configuration <img class="mwe-math-fallback-image-inline tex" alt="T_{VF} = \{\mathbf{T}_{AF} : A,F\in V\}" src="_images/math/f/c/e/fced653d5b38b336f99845bf3502c302.png" /> with <img class="mwe-math-fallback-image-inline tex" alt="V" src="_images/math/5/2/0/5206560a306a2e085a437fd258eb57ce.png" /> being the set of all views and <img class="mwe-math-fallback-image-inline tex" alt="F" src="_images/math/8/0/0/800618943025315f869e4e1f09471012.png" /> being a fixed view that defines the common reference frame.  Then, the above Equation extends to
</p>
<center><img class="mwe-math-fallback-image-inline tex" alt="&#10;\arg\min_{T_{VF}}\sum_{A\in V\setminus\{F\}}\left(\sum_{B\in V\setminus\{A\}}\left(\sum_{(\vec{a},\vec{b})\in C_{AB}}{\left\|\mathbf{T}_{AF}\vec{a} - \mathbf{T}_{BF}\vec{b}\right\|^2}\right)\right)&#10;" src="_images/math/b/5/1/b5121b74adfdedd4b200e563c21ea2f9.png" /></center>
<p>with <img class="mwe-math-fallback-image-inline tex" alt="C_{AB}" src="_images/math/b/3/c/b3cb4b586ee2f02e87e178b6a9514e4a.png" /> being the set of bead correspondences <img class="mwe-math-fallback-image-inline tex" alt="(\vec{a},\vec{b})" src="_images/math/5/6/a/56a686235b248aac541bcbbb94052fd9.png" /> between view&#160;<img class="mwe-math-fallback-image-inline tex" alt="A" src="_images/math/7/f/c/7fc56270e7a70fa81a5935b72eacbe29.png" /> and view&#160;<img class="mwe-math-fallback-image-inline tex" alt="B" src="_images/math/9/d/5/9d5ed678fe57bcca610140957afab571.png" /> whereas <img class="mwe-math-fallback-image-inline tex" alt="\vec{a}\in A" src="_images/math/6/5/f/65fe6cf8eddc220a26383e1ea963fa19.png" /> and <img class="mwe-math-fallback-image-inline tex" alt="\vec{b}\in B" src="_images/math/c/4/7/c476e095b8e67e79f726c6dfad30a75d.png" />.  This term is solved using an iterative optimization scheme.  In each iteration, the optimal affine transformation&#160;<img class="mwe-math-fallback-image-inline tex" alt="\mathbf{T}_{AF}" src="_images/math/b/b/3/bb32bd64dc018797e9e690cdeef9815d.png" /> for each single view&#160;<img class="mwe-math-fallback-image-inline tex" alt="A\in V\setminus\{F\}" src="_images/math/0/d/a/0da4b52ca4ed7ce0d8b722d10e14897b.png" /> relative to the current configuration of all other views is estimated and applied to all beads in this view.  The scheme terminates on convergence of the overall bead correspondence displacement.  This solution allows us to perform the global optimization with any transformation model in case the microscopy set-up has different properties (e. g. translation<sup id="cite_ref-Preibisch2009b_33-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Preibisch2009b-33">&#91;33&#93;</a></sup>, rigid<sup id="cite_ref-Preibisch2009a_30-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Preibisch2009a-30">&#91;30&#93;</a></sup>).
</p>
<h3><span class="mw-headline" id="Time-lapse_registration">Time-lapse registration</span></h3>
<p>During extended time-lapse imaging, the whole agarose column may move. To compensate the drift, we used the bead-based registration framework to register individual time-points to each other. We select a single view <img class="mwe-math-fallback-image-inline tex" alt="A_t" src="_images/math/d/9/8/d98ba6cada8131f61fa3cec8e0c8518e.png" /> from an arbitrary time-point <img class="mwe-math-fallback-image-inline tex" alt="t" src="_images/math/e/3/5/e358efa489f58062f10dd7316b65649e.png" /> in the middle of the series as reference. Subsequently, we use the stored DoG detections to identify the true corresponding local geometric descriptors for all pairs of views <img class="mwe-math-fallback-image-inline tex" alt="A_t" src="_images/math/d/9/8/d98ba6cada8131f61fa3cec8e0c8518e.png" /> and
<img class="mwe-math-fallback-image-inline tex" alt="A_{a\in{\{1,2, \dots \}\setminus\{t\}}}" src="_images/math/9/c/4/9c4dc6a20e290a3fdd003d6656ebde4e.png" /> and calculate an affine transformation&#160;<img class="mwe-math-fallback-image-inline tex" alt="\mathbf{T}_{A_aA_t}" src="_images/math/8/3/e/83e45f70f15ca53e0b3adbe88c75ea23.png" /> that maps <img class="mwe-math-fallback-image-inline tex" alt="A_a" src="_images/math/4/c/4/4c4268c41b567a28da3e53583aec9548.png" /> to <img class="mwe-math-fallback-image-inline tex" alt="A_t" src="_images/math/d/9/8/d98ba6cada8131f61fa3cec8e0c8518e.png" /> by means of least square bead correspondence displacement. The identified transformation matrices are then applied to all remaining views in the respective time-points resulting in a registered time series (<a href="_images/4/46/ErrorAnalysis.png" class="internal" title="ErrorAnalysis.png">Figure&#160;4a</a>).
</p>
<h3><span class="mw-headline" id="Image_Fusion_and_blending">Image Fusion and blending</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="./File:Blending.jpg" class="image"><img alt="" src="_images/e/e9/Blending.jpg" width="300" height="175" class="thumbimage" /></a>  <div class="thumbcaption"><div class="magnify"><a href="./File:Blending.jpg" class="internal" title="Enlarge"></a></div><b>Figure&#160;3:</b> Image fusion</div></div></div>
<p>The registered views can be combined to create a single isotropic 3d image. An effective fusion algorithm must ensure that each view contributes to the final fused volume only useful sharp image data acquired from the area of the sample close to the detection lens.  Blurred data visible in other overlapping views should be suppressed.  We use Gaussian Filters to approximate the image information at each pixel in the contributing views (<a href="_images/e/e9/Blending.jpg" class="internal" title="Blending.jpg">Figure&#160;3c,f</a>).<sup id="cite_ref-PreiRohlHasa2008_34-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-PreiRohlHasa2008-34">&#91;34&#93;</a></sup>
</p><p>For strongly scattering and absorbing specimen like <i>Drosophila</i> , we typically do not image the entire specimen in each single view, but instead stop at roughly two thirds of its depth as the images become increasingly blurred and distorted. In the reconstructed 3d image, this introduces line artifacts in areas where a view ends abruptly (<a href="_images/e/e9/Blending.jpg" class="internal" title="Blending.jpg">Figure&#160;3a,d</a>). To suppress this effect for the purposes of data display, we apply non-linear blending close to the edges of each border between the views (<a href="_images/e/e9/Blending.jpg" class="internal" title="Blending.jpg">Figure&#160;3b,e</a>).<sup id="cite_ref-Preibisch2009b_33-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Preibisch2009b-33">&#91;33&#93;</a></sup>
</p><p>Precise registration of multi-view data is the prerequisite for multi-view deconvolution of the reconstructed image which can potentially increase the resolution.<sup id="cite_ref-Swoger2007_4-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Swoger2007-4">&#91;4&#93;</a></sup><sup id="cite_ref-Engelbrecht2006_35-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Engelbrecht2006-35">&#91;35&#93;</a></sup><sup id="cite_ref-Verveer2007_36-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Verveer2007-36">&#91;36&#93;</a></sup>
Having sub-resolution fluorescent beads around the sample facilitates the estimation of a spatially dependent point spread function and validates the deconvolution results.
</p>
<h3><span class="mw-headline" id="Strategies_for_bead_removal">Strategies for bead removal</span></h3>
<p>The presence of sub-resolution fluorescent beads used for the registration of the views might interfere with subsequent analysis of the dataset. To computationally remove the beads from each view we compute the average bead shape by adding the local image neighborhood of all true correspondences (beads used for registration) of the respective view. The acquired template is subsequently used to identify other beads; to speed up the detection we compare this template only to all maxima detected by the Difference-of-Gaussian operator during the initial bead segmentation step. These DoG-detections contain all image maxima and therefore all beads of the sample. The beads are then removed by subtracting a normalized, Gaussian-blurred version of the bead template. This method reliably removes beads which are clearly separated from the sample judged by the average intensity in the vicinity of the detected bead. Therefore, some beads that are positioned very close to the sample are not removed as the bead-subtraction would interfere with the samples' intensities.
</p><p>To completely remove all beads from the sample we adapt the intensities of the beads to the imaged sample. Therefore we simply embed beads excitable by a different wavelength then the fluorescent maker in the sample and use a long-pass filter for detection (<a href="_images/7/71/Showcase.jpg" class="internal" title="Showcase.jpg">Figure&#160;6e,g</a>). In such acquisition the intensity of the beads is around 2-4.
</p>
<h2><span class="mw-headline" id="Evaluation">Evaluation</span></h2>
<h3><span class="mw-headline" id="Evaluation_of_the_performance_of_the_bead-based_registration_framework">Evaluation of the performance of the bead-based registration framework</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="./File:ErrorAnalysis.png" class="image"><img alt="" src="_images/4/46/ErrorAnalysis.png" width="300" height="137" class="thumbimage" /></a>  <div class="thumbcaption"><div class="magnify"><a href="./File:ErrorAnalysis.png" class="internal" title="Enlarge"></a></div><b>Figure&#160;4:</b> Analysis of the registration error</div></div></div>
<p>We created a visualization of the optimization procedure. For each view, we display its bounding box and the locations of all corresponding descriptors in a 3d visualization framework<sup id="cite_ref-Bene2007_37-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Bene2007-37">&#91;37&#93;</a></sup>.
Correspondences are color coded logarithmically according to their current displacement ranging from red (&gt;100&#160;px) to green (&lt;1&#160;px). The optimization is initialized with a configuration where the orientation of the views is unknown; all views are placed on top of each other and thus the corresponding descriptor displacement is high (red). As the optimization proceeds, the average displacement decreases (yellow) until convergence at about one pixel average displacement (green) is achieved. <a href="_images/7/78/Supplementary_Video_2_Global_Opt.mov" class="internal" title="Supplementary Video 2 Global Opt.mov">Video&#160;2</a>  shows the optimization progress for an 8 angle acquisition of fixed <i>C.elegans</i> . The outline of the worm forms in the middle (grey), since many worm nuclei were segmented by the DoG detector but discarded during establishment of bead correspondences.
</p><p>The global optimization scheme can be seamlessly applied to tiled multi-view acquisitions. In such a set-up, different parts of a large 3d sample are scanned with a high-magnification lens from multiple angles separately. All such acquisitions can be mixed, discarding all information about their arrangement and the global optimization recovers the correct configuration. An example of such optimization is shown in <a href="_images/e/ef/Supplementary_Video_3_Global_Opt_with_Tiling.mov" class="internal" title="Supplementary Video 3 Global Opt with Tiling.mov">Video&#160;3</a>  containing a fixed <i>Drosophila</i>  embryo imaged from 8 angles across two or three tiles per angle on a single photon confocal microscope with a 40&#215;/0.8 Achroplan objective.
</p><p>To prove the accuracy of the bead-based registration framework, we created a simulated bead-only dataset with beads approximated by a Gaussian filter response with &#963;=1.5&#160;px.  We generated 8 different views related by an approximately rigid affine transformation with isotropic resolution. The reconstruction of this dataset yielded an average error of 0.02&#160;px (<a href="SPIM_Registration_Method.html#Table1">Table&#160;1</a>).
For real-life datasets, the registration typically results in errors of about 1&#160;px or slightly lower (<a href="_images/4/46/ErrorAnalysis.png" class="internal" title="ErrorAnalysis.png">Figure&#160;4b</a>  and <a href="SPIM_Registration_Method.html#Table1">Table&#160;1</a>) where the remaining error is introduced by the localization accuracy of the bead detector and small non-affine disturbances induced by elastic deformation of the agarose.  In the <img class="mwe-math-fallback-image-inline tex" alt="xy" src="_images/math/3/e/4/3e44107170a520582ade522fa73c1d15.png" />-plane of each view, the beads can be localized very precisely, however, alongside <img class="mwe-math-fallback-image-inline tex" alt="z" src="_images/math/f/b/a/fbade9e36a3f36d3d676c1b808451dd7.png" />, the localization accuracy drops due to a lower sampling rate and an asymmetric PSF. This is supported by the dimension dependent error shown in <a href="_images/4/46/ErrorAnalysis.png" class="internal" title="ErrorAnalysis.png">Figure&#160;4c,d</a>.
</p><p><a href="SPIM_Registration_Method.html#Table1">Table&#160;1</a> shows that for the registration we usually use significantly more beads than necessary to solve an affine transformation (4 beads). It is necessary to use many beads for registration due to the following reasons:
</p>
<ul><li> If the overlap between stacks is&#8212;as in many examples shown&#8212;very small, it has to be ensured that in these small overlapping areas there are still enough fiduciary markers to align the stacks. </li>
<li> If two stacks are extensively overlapping, the beads have to be evenly distributed around the sample to ensure an even error distribution throughout the sample. Otherwise, small errors in, for example, the lower left corner do not control the registration error in the upper right corner where there are no fiduciary markers.	</li>
<li> The localization error of the beads is normally distributed as shown in <a href="_images/4/46/ErrorAnalysis.png" class="internal" title="ErrorAnalysis.png">Figure&#160;4c,d</a>. That means, the more beads are included in the registration, the more accurate is the average localization of the beads. Consequently, the affine transformation for each individual stack yields lower residual error the more beads are used.</li></ul>
<p><br />
Due to the optical properties of the sample it might occur that beads are distorted and aberrated and therefore detected in the wrong location. Typically, such beads are excluded as they do not form repeatable descriptors between the different stacks. If the distortions are minor, they will contribute to the residual error of the affine transformation. However, the contribution is small as those beads represent a very small fraction of all beads. This is supported by the maximal transfer error of &#160;1 px of the affine model (see also inset of <a href="_images/b/b7/Intensity_vs_Beads.jpg" class="internal" title="Intensity vs Beads.jpg">Figure&#160;5c,d</a>).
</p><p>Examples of within and across time-points registered time-series of <i>Drosophila</i>  embryonic development are shown in <a href="_images/4/41/Supplementary_Video_4_Drosophila_Gastrulation.mov" class="internal" title="Supplementary Video 4 Drosophila Gastrulation.mov">Video&#160;4</a>  and <a href="_images/7/74/Supplementary_Video_5_Drosophila_Embryogenesis.mov" class="internal" title="Supplementary Video 5 Drosophila Embryogenesis.mov">Video&#160;5</a>. The videos show 3d renderings of the developing embryos expressing His-YFP in all cells.  The 3d rendered single embryo is shown from four and three arbitrary angles to highlight the complete coverage of the specimen. <a href="_images/4/41/Supplementary_Video_4_Drosophila_Gastrulation.mov" class="internal" title="Supplementary Video 4 Drosophila Gastrulation.mov">Video&#160;4</a> shows in 42 time-points (7 angles) the last two synchronous nuclear divisions followed by gastrulation. <a href="_images/7/74/Supplementary_Video_5_Drosophila_Embryogenesis.mov" class="internal" title="Supplementary Video 5 Drosophila Embryogenesis.mov">Video&#160;5</a> captures in 249 time-points (5 angles) <i>Drosophila</i>  embryogenesis from gastrulation to mature embryo when muscle activity effectively prevents further imaging.
</p>
<h3><span class="mw-headline" id="Performance_of_the_bead-based_registration_framework_in_comparison_with_intensity-based_methods">Performance of the bead-based registration framework in comparison with intensity-based methods</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="./File:Intensity_vs_Beads.jpg" class="image"><img alt="" src="_images/b/b7/Intensity_vs_Beads.jpg" width="300" height="167" class="thumbimage" /></a>  <div class="thumbcaption"><div class="magnify"><a href="./File:Intensity_vs_Beads.jpg" class="internal" title="Enlarge"></a></div><b>Figure&#160;5:</b> Comparison of bead-based and intensity-based multi-view reconstruction on 7-view acquisition of Drosophila embryo expressing His-YFP</div></div></div>
<p>Existing muti-view SPIM registration approaches that use sample intensities to iteratively optimize the quality of the overlap of the views do not work reliably and are computationally demanding.<sup id="cite_ref-Swoger2007_4-2" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Swoger2007-4">&#91;4&#93;</a></sup><sup id="cite_ref-PreiRohlHasa2008_34-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-PreiRohlHasa2008-34">&#91;34&#93;</a></sup><sup id="cite_ref-Preibisch08a_38-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Preibisch08a-38">&#91;38&#93;</a></sup>
Alternatively, the registration can be achieved by matching of segmented structures, such as cell nuclei, between views<sup id="cite_ref-Keller08_39-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Keller08-39">&#91;39&#93;</a></sup>
However, such approaches are not universally applicable, as the segmentation process has to be adapted to the imaged sample.
</p><p>To evaluate the precision and performance of the bead-based registration framework we compared it against the intensity-based registration method that we developed previously.<sup id="cite_ref-Preibisch08a_38-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Preibisch08a-38">&#91;38&#93;</a></sup>
This method identifies the rotation axis common to all views by iterative optimization of FFT-based phase correlation between adjacent views. We applied both methods (bead-based and intensity-based) to a single time-point of live 7-view acquisition of <i>Drosophila</i>  embryo expressing His-YFP in all cells embedded in agarose with beads. We chose a time-point during blastoderm stage where the morphology of the embryo changes minimally over time. We evaluated the precision of both methods by the average displacement of the corresponding beads and concluded that the bead-based registration framework clearly outperformed the intensity-based registration in terms of bead registration accuracy (0.98&#160;px versus 6.91&#160;px, see <a href="SPIM_Registration_Method.html#Table1">Table&#160;1</a>, <a href="_images/b/b7/Intensity_vs_Beads.jpg" class="internal" title="Intensity vs Beads.jpg">Figure&#160;5</a>). The increased precision in the bead alignment achieved by the bead-based registration framework is reflected in noticeably improved overlap of the nuclei in the sample (see <a href="_images/b/b7/Intensity_vs_Beads.jpg" class="internal" title="Intensity vs Beads.jpg">Figure&#160;5ch</a>). Moreover, the intensity-based method required approximately 9 hours of computation time compared to 2.5&#160;minutes for the bead-based registration framework executed on the same computer hardware (Intel Xeon E5440 with 64GB of RAM), i.e. the bead-based framework is about 200&#160; faster for this dataset.
</p>
<hr />
<table id="Table1" style="text-align:center">
<caption style="margin-top:1em; margin-bottom:1em;"><b>Table&#160;1:</b> Statistics of multi-view registration of various datasets
</caption>
<tr>
<th> Dataset
</th>
<th> min/avg/max error [px]
</th>
<th> DoG detections
</th>
<th> True correspondence number (ratio)
</th>
<th> processing time [min:sec]
</th></tr>
<tr>
<td> Fixed <i>C.elegans</i>, 8 views<br />SPIM 40&#215;/0.8NA
</td>
<td> 1.02/1.12/1.31
</td>
<td> 4566
</td>
<td> 1717 (98%)
</td>
<td> 11:09
</td></tr>
<tr>
<td> Live <i>Drosophila</i>, 5 views<br />SPIM 20&#215;/0.5NA
</td>
<td> 0.76/0.81/1.31
</td>
<td> 9267
</td>
<td> 1459 (97%)
</td>
<td> 2:31
</td></tr>
<tr>
<td> Fixed <i>Drosophila</i>, 10 views<br />SPIM 20&#215;/0.5NA
</td>
<td> 0.65/0.78/0.97
</td>
<td> 9035
</td>
<td> 1301 (93%)
</td>
<td> 20:10
</td></tr>
<tr>
<td> Fixed <i>Drosophila</i>, 11 views<br />Spinning Disc 20&amp;times/0.5NA
</td>
<td> 1.10/1.33/1.86
</td>
<td> 6309
</td>
<td> 978 (92%)
</td>
<td> 6:15
</td></tr>
<tr>
<td> Simulated Dataset, 8 views<br />Isotropic Resolution
</td>
<td> 0.02/0.02/0.02
</td>
<td> 2594
</td>
<td> 2880 (96%)
</td>
<td> 15:54
</td></tr>
<tr>
<td> Live <i>Drosophila</i>, 7 views<br />SPIM 20&#215;/0.5NA<br /><b>bead-based</b>
</td>
<td> 0.87/0.98/1.17
</td>
<td> 6232
</td>
<td> 603 (97%)
</td>
<td> 2:27
</td></tr>
<tr>
<td> Live <i>Drosophila</i>, 7 views<br />SPIM 20&#215;/0.5NA<br /><b>intensity-based</b>
</td>
<td> 0.93/6.91/9.59
</td>
<td> n.a.
</td>
<td> n.a.
</td>
<td> 515:10
</td></tr></table>
<p>We show minimal, average and maximal displacement of all true correspondences (beads) after convergence of the global optimization. The simulated dataset shows very low registration errors. The total number of DoG detections is typically much higher than the number of extracted correspondence candidate pairs although a DoG detection can participate in more than one correspondence pair. The ratio of true correspondences versus correspondence candidates is typically above 90%.  Lower ratios indicate a registration problem, for example caused by movement of the agarose during stack acquisition. The processing time (segmentation and registration) was measured on a dual quad-core Intel Xeon E5440 system.
</p>
<hr />
<h2><span class="mw-headline" id="Samples_imaged_by_SPIM">Samples imaged by SPIM</span></h2>
<h3><span class="mw-headline" id="Overview_of_imaged_specimens">Overview of imaged specimens</span></h3>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="./File:Showcase.jpg" class="image"><img alt="" src="_images/7/71/Showcase.jpg" width="300" height="377" class="thumbimage" /></a>  <div class="thumbcaption"><div class="magnify"><a href="./File:Showcase.jpg" class="internal" title="Enlarge"></a></div><b>Figure&#160;6:</b> Examples of various reconstructed multi-view datasets</div></div></div>
<p>We demonstrated the performance of our registration framework on multi-view <i>in toto</i>  imaging of fixed and living specimen of various model organisms (<a href="_images/7/71/Showcase.jpg" class="internal" title="Showcase.jpg">Figure&#160;6</a>), in particular <i>Drosophila</i> . Fixed <i>Drosophila</i>  embryos were stained with Sytox-Green to label all nuclei. For live imaging, we used a developing <i>Drosophila</i>  embryo expressing fluorescent His-YFP under the control of endogenous promoter visualizing all nuclei. <i>Drosophila</i>  specimens were imaged with a SPIM prototype equipped with a Zeiss 20&#215;/0.5 Achroplan objective.
</p>
<h3><span class="mw-headline" id="Sample_mounting_for_SPIM">Sample mounting for SPIM</span></h3>
<p>We matched the fluorescence intensity of the beads to the signal intensity of the sample. For live imaging of His-YFP which is relatively dim and requires longer exposure times (0.3&#160;s), we used red or yellow fluorescent beads that are suboptimal for the GFP detection filter set and therefore typically less bright than the sample. Conversely, for the imaging of the bright, fixed specimen, we used green fluorescent beads which give adequate signal at very short exposure times (0.01&#160;s).
</p><p>Despite the fact that our algorithm is robust with respect to the amount of beads available for registration, too many beads unnecessarily increase the computation time, while too few beads may result in an inadequate number of correspondences due to incomplete overlap of the views. Therefore, we determined empirically the optimal concentration of beads for each magnification (ideally 1,000&#8211;2,000 beads per imaged volume).  We prepared a 2&#215; stock solution of beads (13&#160;l of concentrated bead solution (Estapor Microspheres FXC050))
</p>
<h3><span class="mw-headline" id="Sample_independent_registration">Sample independent registration</span></h3>
<p>We applied the bead-based registration framework to various samples derived from major model organisms. These include <i>Drosophila</i>  embryo, larva (<a href="_images/7/71/Showcase.jpg" class="internal" title="Showcase.jpg">Figure&#160;6ac</a>) and oogenesis (<a href="_images/7/71/Showcase.jpg" class="internal" title="Showcase.jpg">Figure&#160;6d</a>), <i>C. elegans</i>  adult (data not shown), larval stages (<a href="_images/7/71/Showcase.jpg" class="internal" title="Showcase.jpg">Figure&#160;6e</a>) and early embryo (<a href="_images/7/71/Showcase.jpg" class="internal" title="Showcase.jpg">Figure&#160;6f</a>), whole mouse embryo (<a href="_images/7/71/Showcase.jpg" class="internal" title="Showcase.jpg">Figure&#160;6g</a>), and dual color imaging of zebrafish embryo (<a href="_images/7/71/Showcase.jpg" class="internal" title="Showcase.jpg">Figure&#160;6h</a>). Despite the fact that the samples range significantly in their size, fluorescent labeling, optical properties and mounting formats the bead-based registration framework was invariably capable of achieving the registration. Therefore, we conclude that our method is sample independent and is universally applicable for registration of any multi-view SPIM acquisition where the sample movement does not disturb the rigidity of the agarose.
</p><p><br />
</p>
<h2><span class="mw-headline" id="Broad_applicability_of_the_bead-based_framework_to_multi-view_imaging">Broad applicability of the bead-based framework to multi-view imaging</span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="./File:Rotation_chamber.png" class="image"><img alt="" src="_images/c/c3/Rotation_chamber.png" width="300" height="124" class="thumbimage" /></a>  <div class="thumbcaption"><div class="magnify"><a href="./File:Rotation_chamber.png" class="internal" title="Enlarge"></a></div><b>Figure&#160;7:</b> Multi-view imaging with spinning disc confocal microscopy</div></div></div>
<p>Having the bead-based registration framework for multi-view reconstruction established, we sought to expand its application beyond SPIM, to other microscopy techniques capable of multi-view acquisition.<sup id="cite_ref-Bradl92_40-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-Bradl92-40">&#91;40&#93;</a></sup>
We designed a sample-mounting set-up that allows imaging of a sample embedded in a horizontally positioned agarose column with fluorescent beads (<a href="_images/c/c3/Rotation_chamber.png" class="internal" title="Rotation chamber.png">Figure&#160;7a</a>). The agarose column was manually rotated mimicking the SPIM multi-view acquisition. We acquired multiple views of fixed Drosophila embryos stained with nuclear dye on a spinning disc confocal microscope and reconstructed the views using the bead-based registration framework. By mosaicking around the sample, we captured the specimen in toto and achieved full lateral resolution in areas that are compromised by the poor axial resolution of a single-view confocal stack (<a href="_images/c/c3/Rotation_chamber.png" class="internal" title="Rotation chamber.png">Figure&#160;7b,c,d</a> and <a href="_images/3/34/Supplementary_Video_6_Drosophila_Spinning_Disc.mov" class="internal" title="Supplementary Video 6 Drosophila Spinning Disc.mov">Video&#160;6</a>  ). The combination of multi-view acquisition and bead-based registration is applicable to any imaging modality as long as the fluorescent beads can be localized and the views overlap.
</p>
<h3><span class="mw-headline" id="Sample_mounting_for_multi-view_imaging_on_an_upright_microscope">Sample mounting for multi-view imaging on an upright microscope</span></h3>
<p>We constructed a sample chamber for multi-view imaging on an upright microscope that consists of a teflon dish equipped with a hole in the side wall which has the diameter of a standard glass capillary (<a href="_images/c/c3/Rotation_chamber.png" class="internal" title="Rotation chamber.png">Figure&#160;7a</a>). The capillary mounting hole continues on the bottom of the dish as a semi-circular trench of approximately half the thickness of the capillary diameter, extending about 2/3 of the dish radius towards the center of the dish. This trench serves as a bed for the glass capillary inserted through the capillary mounting hole. The trench is extended by a second, shallower trench whose bottom is elevated with respect to the deeper trench by the thickness of the capillary glass wall. The second trench serves as a bed for the agarose column which is pushed out of the capillary by a tightly fitted plunger (not shown). The teflon dish is equipped on one side with a plastic window enabling visual inspection of the sample and the objective lens.
</p><p>For imaging, the capillary with the sample embedded in agarose containing appropriate amount of fluorescent beads was inserted into the capillary mounting hole until it reached the end of the capillary bed. The teflon dish was filled with water and the agarose was pushed out of the capillary into the agarose bed by the plunger. Water dipping objective was lowered into the dish and focussed on the <i>Drosophila</i>  embryo specimen in agarose. A confocal stack was acquired using variety of optical sectioning techniques (spinning disc confocal (<a href="_images/c/c3/Rotation_chamber.png" class="internal" title="Rotation chamber.png">Figure&#160;7</a>), single photon confocal (see <a href="_images/e/ef/Supplementary_Video_3_Global_Opt_with_Tiling.mov" class="internal" title="Supplementary Video 3 Global Opt with Tiling.mov">Video&#160;3</a>), two photon confocal, apotome (data not shown). Next, to achieve multi-view acquisition, the agarose column was retracted into the capillary by the plunger and the capillary was manually rotated. The angle of the rotation was only very roughly estimated by the position of a tape piece attached to the capillary. The agarose was again pushed out into the agarose bed and another confocal stack was collected. In this way arbitrary number of views can be collected as long as the sample does not bleach.
</p>
<h2><span class="mw-headline" id="Implementation">Implementation</span></h2>
<div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="./File:Screenshot.png" class="image"><img alt="" src="_images/e/e6/Screenshot.png" width="300" height="169" class="thumbimage" /></a>  <div class="thumbcaption"><div class="magnify"><a href="./File:Screenshot.png" class="internal" title="Enlarge"></a></div><b>Figure&#160;8:</b> Screenshot of SPIM registration plugin in Fiji</div></div></div>
<p>The bead-based registration framework is implemented in the Java programming language and provided as a fully open source plugin packaged with the ImageJ distribution <a href="Fiji" title="Fiji">Fiji</a> (Fiji Is Just ImageJ, that is actively developed by an international group of developers. The plugin (<a href="_images/e/e6/Screenshot.png" class="internal" title="Screenshot.png">Figure&#160;8</a>) performs all steps of the registration pipeline: bead segmentation, correspondence analysis of bead-descriptors, outlier removal (RANSAC and global regression), global optimization including optional visualization, several methods for fusion, blending and time-lapse registration.
</p><p>The tutorial on how to use the plugin in basic and advanced mode is available at <a href="SPIM_Registration" title="SPIM Registration">SPIM Registration</a>. The test data containing 7-view SPIM acquisitions of <i>Drosophila</i>  embryo can be downloaded from <a rel="nofollow" class="external autonumber" href="http://fly.mpi-cbg.de/preibisch/nm/HisYFP-SPIM.zip">[1]</a>.
</p>
<h2><span class="mw-headline" id="Acknowledgments">Acknowledgments</span></h2>
<p>We want to thank <a rel="nofollow" class="external text" href="http://www.zeiss.de/micro">Carl Zeiss Microimaging</a> for access to the SPIM demonstrator, Radoslav Kamil Ejsmont<sup id="cite_ref-MPI-CBG_41-0" class="reference"><a href="SPIM_Registration_Method.html#cite_note-MPI-CBG-41">&#91;A 1&#93;</a></sup> for His-YFP flies, Dan White,<sup id="cite_ref-MPI-CBG_41-1" class="reference"><a href="SPIM_Registration_Method.html#cite_note-MPI-CBG-41">&#91;A 1&#93;</a></sup>, Jonathan Rodenfels,<sup id="cite_ref-MPI-CBG_41-2" class="reference"><a href="SPIM_Registration_Method.html#cite_note-MPI-CBG-41">&#91;A 1&#93;</a></sup> Ivana Viktorinova,<sup id="cite_ref-MPI-CBG_41-3" class="reference"><a href="SPIM_Registration_Method.html#cite_note-MPI-CBG-41">&#91;A 1&#93;</a></sup> Mihail Sarov,<sup id="cite_ref-MPI-CBG_41-4" class="reference"><a href="SPIM_Registration_Method.html#cite_note-MPI-CBG-41">&#91;A 1&#93;</a></sup> Steffen Jnsch,<sup id="cite_ref-MPI-CBG_41-5" class="reference"><a href="SPIM_Registration_Method.html#cite_note-MPI-CBG-41">&#91;A 1&#93;</a></sup> Jeremy Pulvers,<sup id="cite_ref-MPI-CBG_41-6" class="reference"><a href="SPIM_Registration_Method.html#cite_note-MPI-CBG-41">&#91;A 1&#93;</a></sup> and Pedro Campinho<sup id="cite_ref-MPI-CBG_41-7" class="reference"><a href="SPIM_Registration_Method.html#cite_note-MPI-CBG-41">&#91;A 1&#93;</a></sup> for providing various biological samples for imaging with SPIM shown in <a href="_images/7/71/Showcase.jpg" class="internal" title="Showcase.jpg">Figure&#160;6</a>.
</p>
<ol class="references">
<li id="cite_note-MPI-CBG-41"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-MPI-CBG_41-0">1.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-MPI-CBG_41-1">1.1</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-MPI-CBG_41-2">1.2</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-MPI-CBG_41-3">1.3</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-MPI-CBG_41-4">1.4</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-MPI-CBG_41-5">1.5</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-MPI-CBG_41-6">1.6</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-MPI-CBG_41-7">1.7</a></sup></span> <span class="reference-text">Max Planck Institute of Molecular Cell Biology and Genetics, Dresden, Germany</span>
</li>
</ol>
<h2><span class="mw-headline" id="References">References</span></h2>
<ol class="references">
<li id="cite_note-HuiskenAl2004-1"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-HuiskenAl2004_1-0">1.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-HuiskenAl2004_1-1">1.1</a></sup></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Optical+Sectioning+Deep+Inside+Live+Embryos+by+Selective+Plane+Illumination+Microscopy&amp;rft.aulast=J.+Huisken+and+J.+Swoger+and+F.+D.+Bene+and+J.+Wittbrodt+and+E.+H.+K.+Stelzer&amp;rft.date=2004&amp;rft.volume=305&amp;rft.issue=5686&amp;rft.pages=1007%26nbsp%3B1010"><cite id="CITEREFJ._Huisken_and_J._Swoger_and_F._D._Bene_and_J._Wittbrodt_and_E._H._K._Stelzer2004">J. Huisken and J. Swoger and F. D. Bene and J. Wittbrodt and E. H. K. Stelzer&#32;(2004),&#32;"Optical Sectioning Deep Inside Live Embryos by Selective Plane Illumination Microscopy",&#32;<i>Science</i>&#32;<b>305</b>&#xa0;(5686):  1007&#160;1010</cite></span>
</span>
</li>
<li id="cite_note-ShawAl89-2"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-ShawAl89_2-0">2.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-ShawAl89_2-1">2.1</a></sup></span> <span class="reference-text">
<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Tilted+view+reconstruction+in+optical+microscopy.+Three-dimensional+reconstruction+of+Drosophila+melanogaster+embryo+nuclei&amp;rft.aulast=P.+Shaw%2C+D.+Agard%2C+Y.+Hiraoka%2C+and+J.+Sedat&amp;rft.date=1989&amp;rft.volume=55&amp;rft.issue=1&amp;rft.pages=101%26ndash%3B110"><cite id="CITEREFP._Shaw.2C_D._Agard.2C_Y._Hiraoka.2C_and_J._Sedat1989">P. Shaw, D. Agard, Y. Hiraoka, and J. Sedat&#32;(1989),&#32;"Tilted view reconstruction in optical microscopy. Three-dimensional reconstruction of Drosophila melanogaster embryo nuclei",&#32;<i>Biophysical Journal</i>&#32;<b>55</b>&#xa0;(1):  101&#8211;110</cite></span>
</span>
</li>
<li id="cite_note-Dodt2007-3"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Dodt2007_3-0"></a></span> <span class="reference-text">
<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Ultramicroscopy%3A+three-dimensional+visualization+of+neuronal+networks+in+the+whole+mouse+brain&amp;rft.aulast=H.+U.+Dodt%2C+U.+Leischner%2C+A.+Schierloh%2C+N.+J%C3%A4hrling%2C+C.+P.+Mauch%2C+K.+Deininger%2C+J.+M.+Deussing%2C+M.+Eder%2C+W.+Zieglg%C3%A4nsberger%2C+and+K.+Becker&amp;rft.date=2007&amp;rft.volume=4&amp;rft.issue=4&amp;rft.pages=331%26ndash%3B336&amp;rft_id=info:doi/10.1038%2Fnmeth1036"><cite id="CITEREFH._U._Dodt.2C_U._Leischner.2C_A._Schierloh.2C_N._J.C3.A4hrling.2C_C._P._Mauch.2C_K._Deininger.2C_J._M._Deussing.2C_M._Eder.2C_W._Zieglg.C3.A4nsberger.2C_and_K._Becker2007">H. U. Dodt, U. Leischner, A. Schierloh, N. Jhrling, C. P. Mauch, K. Deininger, J. M. Deussing, M. Eder, W. Zieglgnsberger, and K. Becker&#32;(2007),&#32;"<a rel="nofollow" class="external text" href="http://dx.doi.org/10.1038%2Fnmeth1036">Ultramicroscopy: three-dimensional visualization of neuronal networks in the whole mouse brain</a>",&#32;<i>Nature Methods</i>&#32;<b>4</b>&#xa0;(4):  331&#8211;336<span class="printonly">, DOI 10.1038/nmeth1036</span></cite></span>
</span>
</li>
<li id="cite_note-Swoger2007-4"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-Swoger2007_4-0">4.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-Swoger2007_4-1">4.1</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-Swoger2007_4-2">4.2</a></sup></span> <span class="reference-text">
<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Multi-view+image+fusion+improves+resolution+in+three-dimensional+microscopy&amp;rft.aulast=J.+Swoger%2C+P.+Verveer%2C+K.+Greger%2C+J.+Huisken%2C+and+E.+H.+K.+Stelzer&amp;rft.date=2007&amp;rft.volume=15&amp;rft.issue=13&amp;rft.pages=8029%26ndash%3B8042"><cite id="CITEREFJ._Swoger.2C_P._Verveer.2C_K._Greger.2C_J._Huisken.2C_and_E._H._K._Stelzer2007">J. Swoger, P. Verveer, K. Greger, J. Huisken, and E. H. K. Stelzer&#32;(2007),&#32;"Multi-view image fusion improves resolution in three-dimensional microscopy",&#32;<i>Opt. Express</i>&#32;<b>15</b>&#xa0;(13):  8029&#8211;8042</cite></span>
 <span class="error mw-ext-cite-error" lang="en" dir="ltr">Cite error: Invalid <code>&lt;ref&gt;</code> tag; name "Swoger2007" defined multiple times with different content</span></span>
</li>
<li id="cite_note-Gullekson1974-5"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Gullekson1974_5-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Three+Dimensional+X-Ray+Opaque+Foreign+Body+Marker+Device&amp;rft.aulast=E.+H.+Gullekson&amp;rft.date=Patent+number%3A+3836776%2C+Filing+date%3A+1+Mar+1973%2C+Issue+date%3A+Sep+1974"><cite id="CITEREFE._H._GulleksonPatent_number:_3836776.2C_Filing_date:_1_Mar_1973.2C_Issue_date:_Sep_1974">E. H. Gullekson&#32;(Patent number: 3836776, Filing date: 1 Mar 1973, Issue date: Sep 1974),&#32;<i>Three Dimensional X-Ray Opaque Foreign Body Marker Device</i></cite></span></span>
</li>
<li id="cite_note-Erickson1993-6"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Erickson1993_6-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Correlation+of+single+photon+emission+CT+with+MR+image+data+using+fiduciary+markers&amp;rft.aulast=B.J.++Erickson+and+Jack+C.R.+Jr.&amp;rft.date=1993&amp;rft.volume=14&amp;rft.issue=3&amp;rft.pages=713%26nbsp%3B720"><cite id="CITEREFB.J._Erickson_and_Jack_C.R._Jr.1993">B.J.  Erickson and Jack C.R. Jr.&#32;(1993),&#32;"Correlation of single photon emission CT with MR image data using fiduciary markers",&#32;<i>American Journal of Neuroradiology</i>&#32;<b>14</b>&#xa0;(3):  713&#160;720</cite></span>
</span>
</li>
<li id="cite_note-Fitzpatrick2001-7"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-Fitzpatrick2001_7-0">7.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-Fitzpatrick2001_7-1">7.1</a></sup></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=The+distribution+of+target+registration+error+in+rigid-body+point-based+registration&amp;rft.aulast=J.+M.+Fitzpatrick+and+J.+B.+West&amp;rft.date=September+2001&amp;rft.volume=20&amp;rft.issue=9&amp;rft.pages=917%26nbsp%3B927"><cite id="CITEREFJ._M._Fitzpatrick_and_J._B._West2001">J. M. Fitzpatrick and J. B. West&#32;(September 2001),&#32;"The distribution of target registration error in rigid-body point-based registration",&#32;<i>IEEE Transactions on Medical Imaging</i>&#32;<b>20</b>&#xa0;(9):  917&#160;927</cite></span>
</span>
</li>
<li id="cite_note-Wiles2008-8"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Wiles2008_8-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=A+Statistical+Model+for+Point-Based+Target+Registration+Error+With+Anisotropic+Fiducial+Localizer+Error&amp;rft.aulast=A.+D.+Wiles%2C+A.+Likholyot%2C+D.+D.+Frantz%2C+and+T.M.Peters&amp;rft.date=March+2008&amp;rft.volume=27&amp;rft.issue=3&amp;rft.pages=378%26nbsp%3B390"><cite id="CITEREFA._D._Wiles.2C_A._Likholyot.2C_D._D._Frantz.2C_and_T.M.Peters2008">A. D. Wiles, A. Likholyot, D. D. Frantz, and T.M.Peters&#32;(March 2008),&#32;"A Statistical Model for Point-Based Target Registration Error With Anisotropic Fiducial Localizer Error",&#32;<i>IEEE Transactions on Medical Imaging</i>&#32;<b>27</b>&#xa0;(3):  378&#160;390</cite></span>
</span>
</li>
<li id="cite_note-Moghari2008-9"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Moghari2008_9-0"></a></span> <span class="reference-text"><cite style="font-style:normal">M. H. Moghari, B. Ma, and P. Abolmaesumi&#32;(2008). "A Theoretical Comparison of Different Target Registration Error Estimators".&#32;<i>MICCAI '08: Proceedings of the 11th International Conference on Medical Image Computing and Computer-Assisted Intervention, Part II</i>: 1032&#160;1040, Berlin, Heidelberg:&#32;Springer-Verlag. doi:<a rel="nofollow" class="external text" href="http://dx.doi.org/10.1007/978-3-540-85990-1_124">10.1007/978-3-540-85990-1_124</a>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=MICCAI+%2708%3A+Proceedings+of+the+11th+International+Conference+on+Medical+Image+Computing+and+Computer-Assisted+Intervention%2C+Part+II&amp;rft.atitle=A+Theoretical+Comparison+of+Different+Target+Registration+Error+Estimators&amp;rft.au=M.+H.+Moghari%2C+B.+Ma%2C+and+P.+Abolmaesumi&amp;rft.date=2008&amp;rft.pub=Springer-Verlag&amp;rft.place=Berlin%2C+Heidelberg&amp;rft.pages=1032%26nbsp%3B1040&amp;rft_id=info:doi/10.1007%2F978-3-540-85990-1_124"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Dierksen1992-10"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Dierksen1992_10-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Towards+automatic+electron+tomography&amp;rft.aulast=K.+Dierksen%2C+D.+Typke%2C+R.+Hegerl%2C+A.+J.+Koster%2C+and+W.+Baumeister&amp;rft.date=1992&amp;rft.volume=40&amp;rft.issue=1&amp;rft.pages=71%26nbsp%3B87"><cite id="CITEREFK._Dierksen.2C_D._Typke.2C_R._Hegerl.2C_A._J._Koster.2C_and_W._Baumeister1992">K. Dierksen, D. Typke, R. Hegerl, A. J. Koster, and W. Baumeister&#32;(1992),&#32;"Towards automatic electron tomography",&#32;<i>Ultramicroscopy</i>&#32;<b>40</b>&#xa0;(1):  71&#160;87</cite></span>
</span>
</li>
<li id="cite_note-Koster1997-11"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Koster1997_11-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Perspectives+of+molecular+and+cellular+electron+tomography&amp;rft.aulast=A.+J.+Koster%2C+R.+Grimm%2C+D.+Typke%2C+R.+Hegerl%2C+A.+Stoschek%2C+J.+Walz%2C+and+W.+Baumeister&amp;rft.date=1997&amp;rft.volume=120&amp;rft.issue=3&amp;rft.pages=276%26nbsp%3B308"><cite id="CITEREFA._J._Koster.2C_R._Grimm.2C_D._Typke.2C_R._Hegerl.2C_A._Stoschek.2C_J._Walz.2C_and_W._Baumeister1997">A. J. Koster, R. Grimm, D. Typke, R. Hegerl, A. Stoschek, J. Walz, and W. Baumeister&#32;(1997),&#32;"Perspectives of molecular and cellular electron tomography",&#32;<i>J Struct Biol</i>&#32;<b>120</b>&#xa0;(3):  276&#160;308</cite></span>
</span>
</li>
<li id="cite_note-HarrisS88-12"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-HarrisS88_12-0"></a></span> <span class="reference-text"><cite style="font-style:normal">C. Harris and M. Stephens&#32;(1988). "A Combined Corner and Edge Detector".&#32;<i>Proceedings of The Fourth Alvey Vision Conference</i>: 147&#160;151.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=Proceedings+of+The+Fourth+Alvey+Vision+Conference&amp;rft.atitle=A+Combined+Corner+and+Edge+Detector&amp;rft.au=C.+Harris+and+M.+Stephens&amp;rft.date=1988&amp;rft.place=Manchester&amp;rft.pages=147%26nbsp%3B151"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Canny86-13"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Canny86_13-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=A+computational+approach+to+edge+detection&amp;rft.aulast=J.+Canny&amp;rft.date=1986&amp;rft.volume=8&amp;rft.issue=6&amp;rft.pages=679%26nbsp%3B698"><cite id="CITEREFJ._Canny1986">J. Canny&#32;(1986),&#32;"A computational approach to edge detection",&#32;<i>IEEE Transactions on Pattern Analysis Machine Intelligence</i>&#32;<b>8</b>&#xa0;(6):  679&#160;698</cite></span>
</span>
</li>
<li id="cite_note-LamdanAl88-14"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-LamdanAl88_14-0">14.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-LamdanAl88_14-1">14.1</a></sup></span> <span class="reference-text"><cite style="font-style:normal">Y. Lamdan, J. Schwartz, and H. Wolfson&#32;(1988). "On recognition of 3D objects from 2D images".&#32;<i>Proceedings of the IEEE International Conference on Robotics and Automation</i>: 1407&#160;1413, Los Alamitos, CA:&#32;IEEE Computer Society.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=Proceedings+of+the+IEEE+International+Conference+on+Robotics+and+Automation&amp;rft.atitle=On+recognition+of+3D+objects+from+2D+images&amp;rft.au=Y.+Lamdan%2C+J.+Schwartz%2C+and+H.+Wolfson&amp;rft.date=1988&amp;rft.pub=IEEE+Computer+Society&amp;rft.place=Los+Alamitos%2C+CA&amp;rft.pages=1407%26nbsp%3B1413"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Hogg2008-15"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Hogg2008_15-0"></a></span> <span class="reference-text"><cite style="font-style:normal">D. W. Hogg, M. Blanton, D. Lang, K. Mierle, and S. Roweis&#32;(August 2008). "Automated Astrometry".&#32;R. W. Argyle, P. S. Bunclark, and J. R. Lewis&#32;<i>Astronomical Data Analysis Software and Systems XVII</i>&#32;<b>394</b>: 27-+.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=Astronomical+Data+Analysis+Software+and+Systems+XVII&amp;rft.atitle=Automated+Astrometry&amp;rft.au=D.+W.+Hogg%2C+M.+Blanton%2C+D.+Lang%2C+K.+Mierle%2C+and+S.+Roweis&amp;rft.date=2008&amp;rft.pages=27-%2B&amp;rft.series=394"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Nussinov91-16"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Nussinov91_16-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Efficient+detection+of+three-dimensional+structural+motifs+in+biological+macromolecules+by+computer+vision+techniques.&amp;rft.aulast=R.+Nussinov+and+H.+J.+Wolfson&amp;rft.date=1991&amp;rft.volume=88&amp;rft.issue=23&amp;rft.pages=10495%26nbsp%3B10499"><cite id="CITEREFR._Nussinov_and_H._J._Wolfson1991">R. Nussinov and H. J. Wolfson&#32;(1991),&#32;"Efficient detection of three-dimensional structural motifs in biological macromolecules by computer vision techniques.",&#32;<i>Proc Natl Acad Sci U S A</i>&#32;<b>88</b>&#xa0;(23):  10495&#160;10499</cite></span>
</span>
</li>
<li id="cite_note-Fischer94-17"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Fischer94_17-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Three-dimensional%2C+sequence+order-independent+structural+comparison+of+a+serine+protease+against+the+crystallographic+database+reveals+active+site+similarities%3A+Potential+implications+to+evolution+and+to+protein+folding&amp;rft.aulast=D.+Fischer%2C+H.+Wolfson%2C+S.+L.+Lin%2C+and+R.+Nussinov&amp;rft.date=1994&amp;rft.volume=3&amp;rft.issue=5&amp;rft.pages=769%26nbsp%3B778"><cite id="CITEREFD._Fischer.2C_H._Wolfson.2C_S._L._Lin.2C_and_R._Nussinov1994">D. Fischer, H. Wolfson, S. L. Lin, and R. Nussinov&#32;(1994),&#32;"Three-dimensional, sequence order-independent structural comparison of a serine protease against the crystallographic database reveals active site similarities: Potential implications to evolution and to protein folding",&#32;<i>Protein Science</i>&#32;<b>3</b>&#xa0;(5):  769&#160;778</cite></span>
</span>
</li>
<li id="cite_note-Wallace1997-18"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Wallace1997_18-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Tess%3A+A+geometric+hashing+algorithm+for+deriving+3D+coordinate+templates+for+searching+structural+databases.+Application+to+enzyme+active+sites&amp;rft.aulast=A.C.+Wallace%2C+N.+Borkakoti%2C+and+J.+M.+Thornton&amp;rft.date=1997&amp;rft.volume=6&amp;rft.issue=11&amp;rft.pages=2308"><cite id="CITEREFA.C._Wallace.2C_N._Borkakoti.2C_and_J._M._Thornton1997">A.C. Wallace, N. Borkakoti, and J. M. Thornton&#32;(1997),&#32;"Tess: A geometric hashing algorithm for deriving 3D coordinate templates for searching structural databases. Application to enzyme active sites",&#32;<i>Protein science</i>&#32;<b>6</b>&#xa0;(11):  2308</cite></span>
</span>
</li>
<li id="cite_note-Stark2003-19"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Stark2003_19-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=A+Model+for+Statistical+Significance+of+Local+Similarities+in+Structure&amp;rft.aulast=A.+Stark%2C+S.+Sunyaev%2C+and+R.+B.+Russell&amp;rft.date=2003&amp;rft.volume=326&amp;rft.issue=5&amp;rft.pages=1307%26nbsp%3B1316"><cite id="CITEREFA._Stark.2C_S._Sunyaev.2C_and_R._B._Russell2003">A. Stark, S. Sunyaev, and R. B. Russell&#32;(2003),&#32;"A Model for Statistical Significance of Local Similarities in Structure",&#32;<i>Journal of Molecular Biology</i>&#32;<b>326</b>&#xa0;(5):  1307&#160;1316</cite></span>
</span>
</li>
<li id="cite_note-Stanski2005-20"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Stanski2005_20-0"></a></span> <span class="reference-text"><cite style="font-style:normal">A. Stanski and O. Hellwich&#32;(2005). "Spiders as Robust Point Descriptors".&#32;<i>DAGM-Symposium</i>: 262&#160;268.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=DAGM-Symposium&amp;rft.atitle=Spiders+as+Robust+Point+Descriptors&amp;rft.au=A.+Stanski+and+O.+Hellwich&amp;rft.date=2005&amp;rft.pages=262%26nbsp%3B268"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Lowe04-21"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-Lowe04_21-0">21.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-Lowe04_21-1">21.1</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-Lowe04_21-2">21.2</a></sup></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Distinctive+image+features+from+scale-invariant+keypoints&amp;rft.aulast=D.+G.+Lowe&amp;rft.date=2004&amp;rft.volume=60&amp;rft.issue=2&amp;rft.pages=91%26nbsp%3B110"><cite id="CITEREFD._G._Lowe2004">D. G. Lowe&#32;(2004),&#32;"Distinctive image features from scale-invariant keypoints",&#32;<i>Int J Comput Vis</i>&#32;<b>60</b>&#xa0;(2):  91&#160;110</cite></span>
</span>
</li>
<li id="cite_note-Kuipers91-22"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Kuipers91_22-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=A+Robot+Exploration+and+Mapping+Strategy+Based+on+a+Semantic+Hierarchy+of+Spatial+Representations&amp;rft.aulast=B.+Kuipers+and+Yung-tai+Byun&amp;rft.date=1991&amp;rft.volume=8&amp;rft.pages=47%26nbsp%3B63"><cite id="CITEREFB._Kuipers_and_Yung-tai_Byun1991">B. Kuipers and Yung-tai Byun&#32;(1991),&#32;"A Robot Exploration and Mapping Strategy Based on a Semantic Hierarchy of Spatial Representations",&#32;<i>Journal of Robotics and Autonomous Systems</i>&#32;<b>8</b>:  47&#160;63</cite></span>
</span>
</li>
<li id="cite_note-Bradley04-23"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Bradley04_23-0"></a></span> <span class="reference-text"><cite style="font-style:normal">D. Bradley, D. Silver, and S. Thayer&#32;(December 2004). "A regional point descriptor for global localization in subterranean environments".&#32;<i>IEEE conference on Robotics Automation and Mechatronics (RAM 2005)</i>&#32;<b>1</b>: 440&#160;445.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=IEEE+conference+on+Robotics+Automation+and+Mechatronics+%28RAM+2005%29&amp;rft.atitle=A+regional+point+descriptor+for+global+localization+in+subterranean+environments&amp;rft.au=D.+Bradley%2C+D.+Silver%2C+and+S.+Thayer&amp;rft.date=2004&amp;rft.pages=440%26nbsp%3B445&amp;rft.series=1"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Frome04-24"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Frome04_24-0"></a></span> <span class="reference-text"><cite style="font-style:normal">A. Frome, D. Huber, R. Kolluri, T. Bulow, and J. Malik&#32;(May 2004). "Recognizing objects in range data using regional point descriptors".&#32;<i>Proceedings of the European Conference on Computer Vision (ECCV)</i>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=Proceedings+of+the+European+Conference+on+Computer+Vision+%28ECCV%29&amp;rft.atitle=Recognizing+objects+in+range+data+using+regional+point+descriptors&amp;rft.au=A.+Frome%2C+D.+Huber%2C+R.+Kolluri%2C+T.+Bulow%2C+and+J.+Malik&amp;rft.date=2004"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Matula2003-25"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-Matula2003_25-0">25.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-Matula2003_25-1">25.1</a></sup></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Precise+3D+image+alignment+in+micro-axial+tomography.&amp;rft.aulast=P.+Matula%2C+M.+Kozubek%2C+F.+Staier%2C+and+M.+Hausmann&amp;rft.date=2003&amp;rft.volume=209&amp;rft.pages=126%26nbsp%3B42"><cite id="CITEREFP._Matula.2C_M._Kozubek.2C_F._Staier.2C_and_M._Hausmann2003">P. Matula, M. Kozubek, F. Staier, and M. Hausmann&#32;(2003),&#32;"Precise 3D image alignment in micro-axial tomography.",&#32;<i>Journal of microscopy</i>&#32;<b>209</b>:  126&#160;42</cite></span>
</span>
</li>
<li id="cite_note-Lindeberg94-26"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Lindeberg94_26-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Scale-space+theory%3A+A+basic+tool+for+analysing+structures+at+different+scales&amp;rft.aulast=T.+Lindeberg&amp;rft.date=1994&amp;rft.volume=21&amp;rft.issue=2&amp;rft.pages=224%26nbsp%3B270"><cite id="CITEREFT._Lindeberg1994">T. Lindeberg&#32;(1994),&#32;"Scale-space theory: A basic tool for analysing structures at different scales",&#32;<i>Journal of Applied Statistics</i>&#32;<b>21</b>&#xa0;(2):  224&#160;270</cite></span>
</span>
</li>
<li id="cite_note-BrownLowe02-27"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-BrownLowe02_27-0"></a></span> <span class="reference-text"><cite style="font-style:normal">M. Brown and D. Lowe&#32;(2002). "Invariant Features from Interest Point Groups".&#32;<i>In British Machine Vision Conference</i>: 656&#160;665.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=In+British+Machine+Vision+Conference&amp;rft.atitle=Invariant+Features+from+Interest+Point+Groups&amp;rft.au=M.+Brown+and+D.+Lowe&amp;rft.date=2002&amp;rft.pages=656%26nbsp%3B665"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-WEKA-28"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-WEKA_28-0">28.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-WEKA_28-1">28.1</a></sup></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Data+Mining%3A+Practical+machine+learning+tools+and+techniques&amp;rft.aulast=I.+H.+Witten+and+E.+Frank&amp;rft.date=2005&amp;rft.edition=second&amp;rft.place=San+Francisco&amp;rft.place=San+Francisco&amp;rft.pub=Morgen+Kaufmann&amp;rft.isbn=0-12-088407-0"><cite id="CITEREFI._H._Witten_and_E._Frank2005">I. H. Witten and E. Frank&#32;(2005),&#32;<i>Data Mining: Practical machine learning tools and techniques</i>&#32;(second ed.), San Francisco: Morgen Kaufmann, ISBN 0-12-088407-0</cite></span>
</span>
</li>
<li id="cite_note-Horn87-29"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Horn87_29-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Closed-form+solution+of+absolute+orientation+using+unit+quaternions&amp;rft.aulast=B.+K.+P.+Horn&amp;rft.date=1987&amp;rft.volume=4&amp;rft.issue=4&amp;rft.pages=629%26nbsp%3B642"><cite id="CITEREFB._K._P._Horn1987">B. K. P. Horn&#32;(1987),&#32;"Closed-form solution of absolute orientation using unit quaternions",&#32;<i>Journal of the Optical Society of America A</i>&#32;<b>4</b>&#xa0;(4):  629&#160;642</cite></span>
</span>
</li>
<li id="cite_note-Preibisch2009a-30"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-Preibisch2009a_30-0">30.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-Preibisch2009a_30-1">30.1</a></sup></span> <span class="reference-text"><cite style="font-style:normal">S.  Preibisch and S.  Saalfeld and T.  Rohlfing and P. Tomancak&#32;(2009). "Bead-based mosaicing of single plane illumination microscopy images using geometric local descriptor matching".&#32;J. P. W.  Pluim and B. M. Dawant&#32;<i>Medical Imaging 2009: Image Processing</i>&#32;<b>7259</b>: 72592S. doi:<a rel="nofollow" class="external text" href="http://dx.doi.org/10.1117/12.812612">10.1117/12.812612</a>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=Medical+Imaging+2009%3A+Image+Processing&amp;rft.atitle=Bead-based+mosaicing+of+single+plane+illumination+microscopy+images+using+geometric+local+descriptor+matching&amp;rft.au=S.++Preibisch+and+S.++Saalfeld+and+T.++Rohlfing+and+P.+Tomancak&amp;rft.date=2009&amp;rft.place=Orlando%2C+FL%2C+USA&amp;rft.pages=72592S&amp;rft.series=7259&amp;rft_id=info:doi/10.1117%2F12.812612"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-FischlerB81-31"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-FischlerB81_31-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Random+sample+consensus%3A+a+paradigm+for+model+fitting+with+applications+to+image+analysis+and+automated+cartography&amp;rft.aulast=M.+A.+Fischler+and+R.+C.+Bolles&amp;rft.date=1981&amp;rft.volume=24&amp;rft.issue=6&amp;rft.pages=381%26nbsp%3B395"><cite id="CITEREFM._A._Fischler_and_R._C._Bolles1981">M. A. Fischler and R. C. Bolles&#32;(1981),&#32;"Random sample consensus: a paradigm for model fitting with applications to image analysis and automated cartography",&#32;<i>Communications of the ACM</i>&#32;<b>24</b>&#xa0;(6):  381&#160;395</cite></span>
</span>
</li>
<li id="cite_note-Hell1993-32"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Hell1993_32-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Aberrations+in+Confocal+Fluorescence+Microscopy+Induced+by+Mismatches+in+Refractive+Index&amp;rft.aulast=S.++Hell+and+G.++Reiner+and+C.++Cremer+and+E.H.K.+Stelzer&amp;rft.date=1993&amp;rft.volume=169&amp;rft.issue=3&amp;rft.pages=341%26nbsp%3B405"><cite id="CITEREFS._Hell_and_G._Reiner_and_C._Cremer_and_E.H.K._Stelzer1993">S.  Hell and G.  Reiner and C.  Cremer and E.H.K. Stelzer&#32;(1993),&#32;"Aberrations in Confocal Fluorescence Microscopy Induced by Mismatches in Refractive Index",&#32;<i>Journal of Microscopy</i>&#32;<b>169</b>&#xa0;(3):  341&#160;405</cite></span>
</span>
</li>
<li id="cite_note-Preibisch2009b-33"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-Preibisch2009b_33-0">33.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-Preibisch2009b_33-1">33.1</a></sup></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Globally+Optimal+Stitching+of+Tiled+3D+Microscopic+Image+Acquisitions&amp;rft.aulast=S.++Preibisch+and+S.++Saalfeld+and+P.+Tomancak&amp;rft.date=2009&amp;rft.volume=25&amp;rft.issue=11&amp;rft.pages=1463%26nbsp%3B1465&amp;rft_id=info:doi/10.1093%2Fbioinformatics%2Fbtp184"><cite id="CITEREFS._Preibisch_and_S._Saalfeld_and_P._Tomancak2009">S.  Preibisch and S.  Saalfeld and P. Tomancak&#32;(2009),&#32;"<a rel="nofollow" class="external text" href="http://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp184">Globally Optimal Stitching of Tiled 3D Microscopic Image Acquisitions</a>",&#32;<i>Bioinformatics</i>&#32;<b>25</b>&#xa0;(11):  1463&#160;1465<span class="printonly">, DOI 10.1093/bioinformatics/btp184</span></cite></span>
</span>
</li>
<li id="cite_note-PreiRohlHasa2008-34"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-PreiRohlHasa2008_34-0">34.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-PreiRohlHasa2008_34-1">34.1</a></sup></span> <span class="reference-text"><cite style="font-style:normal">S.  Preibisch and T.  Rohlfing and M. P.  Hasak and P. Tomancak&#32;(2008). "<a rel="nofollow" class="external text" href="http://link.aip.org/link/?PSI/6914/69140E/1">Mosaicing of Single Plane Illumination Miscroscopy Images Using Groupwise Registration and Fast Content-Based Image Fusion</a>".&#32;J. M.  Reinhardt and J. P. W. Pluim&#32;<i>Medical Imaging 2008: Image Processing</i>&#32;<b>6914</b>: 69140E. doi:<a rel="nofollow" class="external text" href="http://dx.doi.org/10.1117/12.770893">10.1117/12.770893</a>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=Medical+Imaging+2008%3A+Image+Processing&amp;rft.atitle=Mosaicing+of+Single+Plane+Illumination+Miscroscopy+Images+Using+Groupwise+Registration+and+Fast+Content-Based+Image+Fusion&amp;rft.au=S.++Preibisch+and+T.++Rohlfing+and+M.+P.++Hasak+and+P.+Tomancak&amp;rft.date=2008&amp;rft.place=San+Diego%2C+CA%2C+USA&amp;rft.pages=69140E&amp;rft.series=6914&amp;rft_id=info:doi/10.1117%2F12.770893&amp;rft_id=http%3A%2F%2Flink.aip.org%2Flink%2F%3FPSI%2F6914%2F69140E%2F1"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Engelbrecht2006-35"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Engelbrecht2006_35-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Resolution+enhancement+in+a+light-sheet-based+microscope+%28SPIM%29&amp;rft.aulast=C.+J.+Engelbrecht+and+E.+H.+K.+Stelzer&amp;rft.date=2006&amp;rft.volume=31&amp;rft.issue=10&amp;rft.pages=1477%26nbsp%3B1479"><cite id="CITEREFC._J._Engelbrecht_and_E._H._K._Stelzer2006">C. J. Engelbrecht and E. H. K. Stelzer&#32;(2006),&#32;"Resolution enhancement in a light-sheet-based microscope (SPIM)",&#32;<i>Optics  Letters</i>&#32;<b>31</b>&#xa0;(10):  1477&#160;1479</cite></span>
</span>
</li>
<li id="cite_note-Verveer2007-36"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Verveer2007_36-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=High-resolution+three-dimensional+imaging+of+large+specimens+with+light+sheet-based+microscopy&amp;rft.aulast=P.J.+Verveer+and+J.+Swoger+and+F.+Pampaloni+and+K.+Greger+and+M.+Marcello+and+E.H.K.Stelzer&amp;rft.date=2007&amp;rft.volume=4&amp;rft.issue=4&amp;rft.pages=311%26nbsp%3B313"><cite id="CITEREFP.J._Verveer_and_J._Swoger_and_F._Pampaloni_and_K._Greger_and_M._Marcello_and_E.H.K.Stelzer2007">P.J. Verveer and J. Swoger and F. Pampaloni and K. Greger and M. Marcello and E.H.K.Stelzer&#32;(2007),&#32;"High-resolution three-dimensional imaging of large specimens with light sheet-based microscopy",&#32;<i>Nature Methods</i>&#32;<b>4</b>&#xa0;(4):  311&#160;313</cite></span>
</span>
</li>
<li id="cite_note-Bene2007-37"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Bene2007_37-0"></a></span> <span class="reference-text"><cite style="font-style:normal">B. Schmidt&#32;(2008). "Hardware-accelerated 3D visualization for ImageJ".&#32;Pierre Plumer&#32;<i>ImageJ User and Developer Conference</i>&#32;<b>2</b>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=ImageJ+User+and+Developer+Conference&amp;rft.atitle=Hardware-accelerated+3D+visualization+for+ImageJ&amp;rft.au=B.+Schmidt&amp;rft.date=2008&amp;rft.series=2"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Preibisch08a-38"><span class="mw-cite-backlink"> <sup><a href="SPIM_Registration_Method.html#cite_ref-Preibisch08a_38-0">38.0</a></sup> <sup><a href="SPIM_Registration_Method.html#cite_ref-Preibisch08a_38-1">38.1</a></sup></span> <span class="reference-text"><cite style="font-style:normal">S. Preibisch and R. Ejsmont and T. Rohlfing and P. Tomancak&#32;(2008). "Towards Digital Representation of Drosophila Embryogenesis".&#32;<i>Proceedings of 5th IEEE International Symposium on Biomedical Imaging</i>: 324&#160;327.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=proceeding&amp;rft.btitle=Proceedings+of+5th+IEEE+International+Symposium+on+Biomedical+Imaging&amp;rft.atitle=Towards+Digital+Representation+of+Drosophila+Embryogenesis&amp;rft.au=S.+Preibisch+and+R.+Ejsmont+and+T.+Rohlfing+and+P.+Tomancak&amp;rft.date=2008&amp;rft.pages=324%26nbsp%3B327"><span style="display: none;">&#160;</span></span></span>
</li>
<li id="cite_note-Keller08-39"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Keller08_39-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=Reconstruction+of+zebrafish+early+embryonic+development+by+scanned+light+sheet+microscopy&amp;rft.aulast=P.+J.++Keller+and+A.+D.++Schmidt+and+J.++Wittbrodt+and+E.+H.+Stelzer&amp;rft.date=2008&amp;rft.volume=322&amp;rft.issue=5904&amp;rft.pages=1065-9"><cite id="CITEREFP._J._Keller_and_A._D._Schmidt_and_J._Wittbrodt_and_E._H._Stelzer2008">P. J.  Keller and A. D.  Schmidt and J.  Wittbrodt and E. H. Stelzer&#32;(2008),&#32;"Reconstruction of zebrafish early embryonic development by scanned light sheet microscopy",&#32;<i>Science</i>&#32;<b>322</b>&#xa0;(5904):  1065-9</cite></span>
</span>
</li>
<li id="cite_note-Bradl92-40"><span class="mw-cite-backlink"><a href="SPIM_Registration_Method.html#cite_ref-Bradl92_40-0"></a></span> <span class="reference-text"><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.title=A+tilting+device+for+three-dimensional+microscopy%3A+application+to+in+situ+imaging+of+interphase+cell+nuclei.&amp;rft.aulast=J.+Bradl+and+M.+Hausmann+and+V.+Ehemann+and+D.+Komitowski+and+C.+Cremer&amp;rft.date=1992&amp;rft.volume=168&amp;rft.pages=47%26nbsp%3B57"><cite id="CITEREFJ._Bradl_and_M._Hausmann_and_V._Ehemann_and_D._Komitowski_and_C._Cremer1992">J. Bradl and M. Hausmann and V. Ehemann and D. Komitowski and C. Cremer&#32;(1992),&#32;"A tilting device for three-dimensional microscopy: application to in situ imaging of interphase cell nuclei.",&#32;<i>Journal of Microscopy</i>&#32;<b>168</b>:  47&#160;57</cite></span>
</span>
</li>
</ol>

<!-- 
NewPP limit report
Cached time: 20200713073302
Cache expiry: 86400
Dynamic content: false
CPU time usage: 0.448 seconds
Real time usage: 0.457 seconds
Preprocessor visited node count: 13222/1000000
Preprocessor generated node count: 18468/1000000
Postexpand include size: 95511/2097152 bytes
Template argument size: 39330/2097152 bytes
Highest expansion depth: 13/40
Expensive parser function count: 0/3
-->

<!-- 
Transclusion expansion time report (%,ms,calls,template)
100.00%  295.261      1 - -total
 77.71%  229.451     28 - Template:Citation/core
 75.89%  224.059     26 - Template:Cite_journal
 15.66%   46.241     12 - Template:Cite_conference
  4.98%   14.718     28 - Template:Link
  3.78%   11.174      1 - Template:Cite_book
  3.44%   10.148      1 - Template:Citation
-->
</div><div class="printfooter">
Retrieved from "<a dir="ltr" href="index.php?title=SPIM_Registration_Method&amp;oldid=13723">http://imagej.net/index.php?title=SPIM_Registration_Method&amp;oldid=13723</a>"</div>
							</div>

			<div id="footer">
				<p> This page was last modified on 23 December 2014, at 08:32.</p><ul><li><a href="./ImageJ:Privacy_policy" title="ImageJ:Privacy policy">Privacy policy</a></li><li><a href="./ImageJ:About" class="mw-redirect" title="ImageJ:About">About ImageJ</a></li><li><a href="Imprint" title="Imprint">Imprint</a></li></ul>			</div>

			<div id="catlinks" class="catlinks" data-mw="interface"><div id="mw-normal-catlinks" class="mw-normal-catlinks"><a href="./Special:Categories" title="Special:Categories">Categories</a>: <ul><li><a href="index.php?title=Category:Pages_with_reference_errors&amp;action=edit&amp;redlink=1" class="new" title="Category:Pages with reference errors (page does not exist)">Pages with reference errors</a></li><li><a href="./Category:Plugins" title="Category:Plugins">Plugins</a></li><li><a href="./Category:Registration" title="Category:Registration">Registration</a></li></ul></div></div>		</div>

		<div id="bottom-wrap">
		</div>
		<script>(window.RLQ=window.RLQ||[]).push(function(){mw.loader.load(["ext.fancytree","ext.suckerfish","ext.cite.a11y","mediawiki.toc","mediawiki.action.view.postEdit","site","mediawiki.user","mediawiki.hidpi","mediawiki.page.ready","mediawiki.searchSuggest","ext.SimpleTooltip"]);});</script><script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgBackendResponseTime":654});});</script>
		</body>
		</html>
		